rs/execution_environment/src/scheduler.rs

```diff
@@ -1943,6 +1951,7 @@ fn observe_instructions_consumed_per_message(
 struct ExecutionThreadResult {
     canisters: Vec<CanisterState>,
     executed_canister_ids: BTreeSet<CanisterId>,
+    executed_priority_ids: BTreeMap<CanisterId, AccumulatedPriority>,
```

Comment:
Personal opinion: This name is quite opaque. (What's a "priority ID"? Or an "executed priority"?) I also don't think a map is necessary, as the value is always 100 and we only execute a canister once during an iteration.

So I would make it something like `fully_executed_canister_ids: Vec<CanisterId>` (or `full_round_canisters`; or `canisters_with_full_rounds`).

Reply:
Renamed to `fully_executed_canister_ids`, switched to `Vec`. The `AccumulatedPriority` is used in the follow up PR.